hammer/technology/sky130/defaults.yml

```diff
@@ -92,7 +92,7 @@ vlsi:
     tap_cell_interval: 15.0 # Set the interval and offset for tap cells
     tap_cell_offset: 5.0
 
-    routing_layers: [2, 6]
+    routing_layers: [1, 5]
```

Comment:
Are you sure this is right? SKY 130 has li1 to met5, [2, 6] refers to using met1-met5 I believe.

Reply:
I want to say I changed it due to the custom lef mapping, but the change I think this was due to (making li1 masterslice) was reverted. Will change back and make sure `power-par` still runs.